aws-logs-metricfilter/src/main/java/software/amazon/logs/metricfilter/Translator.java

```diff
@@ -5,9 +5,7 @@
 import software.amazon.awssdk.services.cloudwatchlogs.model.ResourceNotFoundException;
 import software.amazon.cloudformation.exceptions.*;
 
-import java.util.Collection;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
```

Comment:
nit: can we avoid using wildcard imports

Reply:
updated!